ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/FilterParameterTest.java

```diff
@@ -224,6 +243,91 @@ public void emptyListAsSubTopicDecodesCorrectly() throws JsonProcessingException
             emptyList());
   }
 
+  @Test
+  public void emptyListOfTopicsDecodesCorrectly() throws java.io.IOException {
+    final FilterParameter filterParameter = readJsonAsFilterParameter("{\"topics\":[]}");
+
+    assertThat(filterParameter.getTopics().size()).isEqualTo(0);
+  }
+
+  @Test
+  public void emptyListOfTopicsNestedDecodesCorrectly() throws java.io.IOException {
+    final FilterParameter filterParameter = readJsonAsFilterParameter("{\"topics\":[[]]}");
+
+    assertThat(filterParameter.getTopics()).containsExactly(List.of());
```

Comment:
nit: _personally_ would prefer to see "emptyList()" ... but maybe that's arbitrary :)